### PR TITLE
Message view

### PR DESCRIPTION
**Pull request by SimonNumberTwo**
# Description
This MR adds a generic message view which can have a contentView embedded. One can configure it to show up from bottom or from top. One can add a background view with a given background color. When tapping on this background view, the message will be hidden. Also when tapping on the message view or swiping it away it disappears.

Furthermore queueing is being implemented so when wanting to show multiple message views, they will show up after each other.

# Testing
For testing purposes the sample project was adjusted.

### CODE REVIEW

**Review by jamitJona — APPROVED** [review]

File: JamitFoundationExample/App/Sources/MessageView/Info/InfoMessageView.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright © 2021 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
```
[the remark this answers is not in the document]
Comment (SimonNumberTwo) [reply JamitFoundationExample/App/Sources/MessageView/Info/InfoMessageView.swift:3]: Done.

File: JamitFoundationExample/App/Sources/MessageView/Info/InfoMessageViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Copyright © 2021 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
```
[the remark this answers is not in the document]
Comment (SimonNumberTwo) [reply JamitFoundationExample/App/Sources/MessageView/Info/InfoMessageViewModel.swift:3]: Done.

File: Modules/MessageView/Presenter/MessageViewPresenter.swift [hunk 3]
```diff
@@ -0,0 +1,188 @@
+// Copyright © 2021 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
```
[the remark this answers is not in the document]
Comment (SimonNumberTwo) [reply Modules/MessageView/Presenter/MessageViewPresenter.swift:3]: Done.

File: Modules/MessageView/Presenter/MessageViewPresenterConfiguration.swift [hunk 4]
```diff
@@ -0,0 +1,81 @@
+//  Copyright © 2021 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
```
[the remark this answers is not in the document]
Comment (SimonNumberTwo) [reply Modules/MessageView/Presenter/MessageViewPresenterConfiguration.swift:3]: Done.

**Pull request merged**